src/components/PhoneCards/PhonesForHotPrices.tsx

```diff
@@ -0,0 +1,208 @@
+import React, { useContext } from 'react';
+import { DispatchContext, StateContext } from '../../Store';
+import { Product } from '../../types';
+import { Link } from 'react-router-dom';
+import stroke from '../../assets/img/icons/Vector (Stroke).svg';
+import { useTheme } from '../../contexts/ThemeContext';
+
+type Props = {
+  image: string;
+  name: string;
+  fullPrice: number;
+  price: number;
+  screen: string;
+  capacity: string;
+  ram: string;
+  product: Product;
+};
+
+export const PhonesForHotPrices: React.FC<Props> = ({
+  image,
+  name,
+  fullPrice,
+  price,
+  screen,
+  capacity,
+  ram,
+  product,
+}) => {
+  const dispatch = useContext(DispatchContext);
+  const state = useContext(StateContext);
+  const { favorites, basket } = state;
+  const { theme } = useTheme();
+
+  const isFavorite = !!favorites.find(item => item === product.itemId);
+  const added = basket.find(
+    (item: { itemId: string }) => item.itemId === product.itemId,
+  );
+
+  const addToFavorites = () => {
+    if (!isFavorite) {
+      dispatch({
+        type: 'addToFavorites',
+        payload: product,
+      });
+    }
+
+    if (isFavorite) {
+      dispatch({
+        type: 'removeFromFavorites',
+        payload: { itemId: product.itemId },
+      });
+    }
+  };
+
+  const addToBascket = () => {
+    if (!added) {
+      dispatch({
+        type: 'addToBasket',
+        payload: product,
+      });
+    }
+
+    if (added) {
+      dispatch({
+        type: 'removeFromBasket',
+        payload: { itemId: product.itemId },
+      });
+    }
+  };
+
+  return theme === 'light' ? (
+    <div className="w-72 shadow-lg rounded-lg p-8 transition-transform duration-500 transform hover:scale-105">
+      <Link to={`/${product.category}/${product.itemId}`}>
+        <img
+          src={`${image}`}
+          alt="img"
+          className="w-full h-56 object-contain rounded-t-lg"
+        />
+      </Link>
+      <Link to={`/${product.category}/${product.itemId}`}>
+        <h3 className="block mt-1 h-12 mb-[-20px] font-medium text-[14px] leading-[21px] text-black text-left">
+          {name}
+        </h3>
+      </Link>
+      <div className="mt-4 w-full flex gap-2">
+        <p className="text-xl font-bold text-black mt-2 text-left mb-3">{`$${price}`}</p>
+        <p className="text-xl font-semibold text-[22px] text-[#89939A] line-through mt-2 text-left">{`$${fullPrice}`}</p>
+      </div>
```

Comment:
We already have a ready-made product card component. Why do we need two more, almost completely duplicating the layout of the already finished component?